### ISSUE TITLE
WebServer on another port than 80

### ISSUE THREAD

**Issue by nik101968** [bricoleau/WiFiTerm#3]
I tried configure WebServer on port 60 for example, WebServer connect but not receive data, arrow button is red

**Comment by SoftVienka** [comment]
That would be good for me too. I connect myself from the outside and use Port 8080. Of course Wifiterm is not working. Probably because routine in the library folds the address for websocket xxx.xxx.xxx.xxx:8080:81, which will not work.

**Comment by buccaneer-jak** [comment]
I agree. Being able to use other ports such as 81 would be a great addition.

**Comment by SoftVienka** [comment]
Now I use `TelnetStream2.h `and `puttytel` on Windows. I still have to find a suitable app for Android.
